lib/ui/semantics.dart

```diff
@@ -307,6 +309,24 @@ class SemanticsFlag {
   /// is a password or contains other sensitive information.
   static const SemanticsFlag isObscured = const SemanticsFlag._(_kIsObscuredIndex);
 
+  /// Whether the semantics node is the root of a subtree for which values
+  /// should be announced.
+  /// 
+  /// When a semantics node with an edge flag is added to the tree, the
+  /// framework will search for a child with a route flag and non-empty
+  /// semantic value and announce it as an edge transition.
```

Comment:
This should say what the exact rule is. Is it depth-first pre-order? post-order? first found? last found?

Reply:
Updated comment to specify order